client/my-sites/upgrades/domain-management/transfer/index.jsx

```diff
@@ -1,84 +1,44 @@
 /**
- * External dependencies
+ * External Dependencies
```

Comment:
I don't think it matters. FWIW, this was not an intentional change, it is just highlighted because I moved transfer to another file but didn't do `git mv`.